### ISSUE TITLE
Unable to remove unique on a field

### ISSUE THREAD

**Issue by JbalTero** [directus/api#1188]
## Steps to Reproduce

* Fresh install of v2.4.0
* Create a collection
* Add a text field with unique enabled
* Modify the text field to disable unique

## Expected Behavior

The unique of field should be removed.

## Actual Behavior

The field is still unique.

## Other Context & Screenshots

[Screen recording](https://drive.google.com/file/d/1vs5w7fXlYhwfXKUFa8HZv0DZU0cgcwQG/view?usp=sharing)

## Technical Details

- Device: Desktop
- OS: Win 10  10.0.18362.145
- Web Server: Apache/2.4.29
- PHP Version: PHP 7.1.30
- Database: 10.2.26
- Install Method: v2.4.0 build
- Directus App: 7.9.0

**Comment by justrealmilk** [comment]
Try updating the field as you did, then refreshing the app before checking if it worked as expected. See what happens

**Comment by JbalTero** [comment]
@justrealmilk I tried your suggestion in DIrectus demo, the same thing happens, it still unique.

**Comment by bjgajjar** [comment]
Hey @JbalTero 

This is a known issue of Zend DB that user is not able to remove the constraint. You can check [here](https://github.com/zendframework/zend-db/issues/36)

@benhaynes 

I think we should update our fork repo of Zend-DB.

**Comment by benhaynes** [comment]
Hey @bjgajjar — any idea how complex this would be to update in our fork? I guess we would need to find out if this is just something they have "turned off" for some reason... or if it's a complex thing to get working.

**Comment by bjgajjar** [comment]
Hey @benhaynes 

> any idea how complex this would be to update in our fork?

It's not that much complex actually. But I need access to the repo. 

> I guess we would need to find out if this is just something they have "turned off" for some reason.

Don't worry. I'll check before update the code.

**Comment by benhaynes** [comment]
Great, thanks @bjgajjar! I have given the API team admin access to the Zend fork.

https://github.com/directus/zend-db

**Comment by bjgajjar** [comment]
@benhaynes - This will be on hold until [PR](https://github.com/zendframework/zend-db/pull/247) from `Zend-DB` is not getting merged.

I recommended some changes there. So we have to wait/develop it.

**Comment by benhaynes** [comment]
Don't we have our own fork of that repo? I thought we did, but maybe I am mistaken.

**Comment by bjgajjar** [comment]
> Don't we have our own fork of that repo?

Yes, we have a forked repo. But as `Zend-DB` don't have this feature yet. We need to develop it by our self in our fork repo.

**Comment by bjgajjar** [comment]
@benhaynes - Should we consider it a known issue or should we try to develop it?

I would like to mention here that we didn't take any decision yet for whether we should use fork repo or `Zend-DB` itself.

**Comment by benhaynes** [comment]
Let's hold on this, I don't think we want to fork Zend... and this _should_ be solved in the Laravel port. @rijkvanzanten @WoLfulus  do you agree?

**Comment by rijkvanzanten** [comment]
Zend-db is archived and no longer maintained, and it's a known issue in there. We rely on zend-db for our database abstraction layer. Once we're moved away from Zend-db, this issue will be resolved too.

**Comment by bjgajjar** [comment]
>Once we're moved away from Zend-db, this issue will be resolved too.

Agree. So should we close this or still need to wait until we moved away from `Zend`?

**Comment by rijkvanzanten** [comment]
Lets leave it open until we've migrated away from Zend, as it won't be fixed until we do
